docs/concepts/services-networking/service.md

```diff
@@ -428,30 +428,26 @@ an ephemeral IP will be assigned to the loadBalancer. If the `loadBalancerIP` is
 cloud provider does not support the feature, the field will be ignored.
 
 #### SSL support on AWS
-For partial SSL support on clusters running on AWS, starting with 1.3 two
-annotations can be added to a `LoadBalancer` service:
+For partial SSL support on clusters running on AWS, the following annotations can be 
```

Comment:
Unclear what the original author meant by "partial." Configuring full SSL to the backend pods is painful and may require work to be done outside of the Kubernetes CLI. Most use an ELB to terminate the SSL connection and forward via HTTP.